test/clj/rems/test/api/applications.clj

```diff
@@ -813,6 +814,73 @@
           "user should be forbidden to send command"))
 
     (testing "send commands with authorized user"
+      (testing "even handler cannot comment without request"
```

Comment:
These tests seem to be mostly duplicating `rems.workflow.dynamic/test-comment`. Integration tests should only check that integration (e.g. HTTP transport layer) is working, but not all the detailed business rules, which are better checked using fast unit tests.